src/ml-service/packages/ml-api/api/predict.py

```diff
@@ -0,0 +1,157 @@
+import numpy as np
```

Comment:
Great work.
Just trying to understand to appreciate the design patterns employed so far as I do a quick run on the changed files.
Under what MVC component would this file of predict.py fall? In case it does not fall under the MVC, under what category would it belong?